htdocs/core/actions_sendmails.inc.php

```diff
@@ -185,12 +185,12 @@
 		}
 
 		$tmparray = array();
-		if (trim($_POST['sendto'])) {
+		if (isset($_POST['sendto'])) {
```

Comment:
I think isset is not enough. We must enter the if if value us not empty too   Why not using

 if GETPOST

Reply:
i post a fix